### ISSUE TITLE
BatchNormalization update ops with gradient tape

### ISSUE THREAD

**Issue by iperov** [tensorflow/tensorflow#32864]
how to update moving mean and moving var in each BatchNormalization layer in gradient tape mode?

**Comment by iperov** [comment]
solved:

batch normalization layer should be called with argument *training=True*
